async.c

```diff
@@ -467,8 +466,8 @@ static int __redisGetSubscribeCallback(redisAsyncContext *ac, redisReply *reply,
         }
         sdsfree(sname);
     } else {
-        /* Shift callback for invalid commands. */
-        __redisShiftCallback(&ac->sub.invalid,dstcb);
+        /* Get callback for pending command during subscribe. */
+        __redisShiftCallback(&ac->sub.replies,dstcb);
```

Comment:
Why replace "Shift" with "Get" in this comment?

Shift means pop from the beginning of the list. This meaning of `shift` is known from shell scripting.